client/src/components/downloadPacket.module.scss

```diff
@@ -0,0 +1,49 @@
+$primary-color: #112f4e;
+
+.downloadBoxContainer {
+    flex: 0 0 350px;
```

Comment:
At some point we do have to break down to pixels, but I'd wonder if we could use percentages and/or column widths from USWDS? `content` may also work for `flex-basis` .

Reply:
good point! I will look into this